visualize-admin/visualization-tool
# Fix/smaller improvements

This PR resolves #1818 , resolves #1800 and resolves #1758
- This PR adds sharing and embedding capabilities directly into the users profile 
- This PR improves the visibiliy current dataset used in during Visualization Creation 
- This PR resolves the overlapping bug when using interactive Filters

### Next Steps
1. @KerstinFaye please provide a design for #1800 and #1758 like this I can proceed. More details about ideas from the Team you can find under the Issue's themselves ✅

---


### How to Test (1800)
1. Go to this [Link](https://vercel.live/open-feedback/visualization-tool-git-fix-smaller-improvements-ixt1.vercel.app?via=pr-comment-visit-preview-link&passThrough=1)
2. Create a Visualization (f.e Einmalvergütung für Photovoltaikanlagen)
3. See how the Data Sources Section is Called Datasets ✅
4. See how the Datasets Section is all the way on the top ✅
5. See how button styling within the Datasets Section changed from outlined to text ✅

---

### How to Test (1818)
1. Go to this [Link](https://vercel.live/open-feedback/visualization-tool-git-fix-smaller-improvements-ixt1.vercel.app?via=pr-comment-visit-preview-link&passThrough=1) 
2. Create a new Visualization 
3. Select "Columns" or "Areas" as the chart type.
4. Add a segmentation column in the Chart options.
5. Change the Chart mode to interactive
6. Observe the overlapping elements.

> Test how it was before 
> old Issue [Link](https://visualize.admin.ch/en/browse?previous=%7B%22type%22%3A%22organization%22%2C%22iri%22%3A%22https%3A%2F%2Fregister.ld.admin.ch%2Fopendataswiss%2Forg%2Fbundesamt-fur-lebensmittelsicherheit-und-veterinaerwesen-blv%22%2C%22topic%22%3A%22%22%7D&dataset=https%3A%2F%2Fhealth.ld.admin.ch%2Ffsvo%2FBT_entwicklung%2F14&dataSource=Prod)


---

### How to Test (1758)
- #1758 can only be tested once deployed, I will be providing a video soon
